src/main/groovy/si/dlabs/gradle/extensions/CheckstyleExtension.groovy

```diff
@@ -7,7 +7,6 @@ import org.gradle.api.tasks.Input
  */
 class CheckstyleExtension extends PluginBaseExtension {
 
-    @Input String reportFile = "reports/checkstyle/checkstyle.xml"
```

Comment:
Because it's unused, see the [commit message](https://github.com/sschuberth/soter/commit/c6034b2c6bb9088903e529f6748c477be8a05e6b).

I highly recommend to review PRs commit by commit to understand the steps and read the commit messages instead of only looking at the combined diff of the PR, even if that's unfortunately the workflow GitHub suggests.